arai60/11-16_HashMap/11_01_TwoSum/level_2.py

```
@@ -0,0 +1,64 @@
+# 総当たりで判定
+# for文はrangeを使う方法に統一
+class Solution:
+    def twoSum(self, nums: List[int], target: int) -> List[int]:
+        for i in range(len(nums) - 1):
+            for j in range(i + 1, len(nums)):
+                if target == nums[i] + nums[j]:
+                    return [i, j]
+        return []
```

課題文では解が保証されていますが、見つからなかったときにどうするかは好みですかね。自分はexception投げちゃいます。変に値を返してしまうと後続の変な場所でバグってしまう恐れがあると思うので。fail fastに近い考えです。